src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -2729,8 +2730,9 @@ private bool IsMisplacedModifier(SyntaxListBuilder modifiers, SyntaxList<Attribu
                     return true;
                 }
 
-                var incompleteMember = _syntaxFactory.IncompleteMember(attributes, modifiers.ToList(), type.IsMissing ? null : type);
-                if (incompleteMember.ContainsDiagnostics)
+                var incompleteMember = _syntaxFactory.FieldDeclaration(attributes, modifiers.ToList(), CreateVariableDeclarationWithTypeAndSingleMissingDeclarator(type), CreateMissingSemicolonToken());
+                //var incompleteMember = _syntaxFactory.IncompleteMember(attributes, modifiers.ToList(), type.IsMissing ? null : type);
+                if (incompleteMember.ContainsDiagnostics && !type.IsMissing)
```

Comment:
>incompleteMember.ContainsDiagnostics && !type.IsMissing [](http://example.com/codeflow?start=20&length=55)

The new condition doesn't look equivalent to the old one. Perhaps we should check if attributes and modifiers have diagnostics and ignore presence of diagnostics from other parts?